src/main/resources/assets/openblocks/lang/de_de.lang

```diff
@@ -99,7 +99,7 @@ openblocks.misc.get_witched=Werde verhext!
 openblocks.misc.page=Seite %d von %d
 openblocks.misc.oh_no_ceiling=Du kannst hier nicht schlafen. Die Decke beunruhigt dich zu sehr...
 openblocks.misc.oh_no_ground=Auf DEM Ding willst du schlafen?!
-openblocks.misc.sleeping_bag_broken=Item inaktiv wegen fehlerhafter Initialisierung
+openblocks.misc.sleeping_bag_broken=Gegenstand inaktiv wegen fehlerhafter Initialisierung
```

Comment:
Stop trying to translate "Item".

Reply:
However :P

Reply:
Yes, that is not what "However" means...